queries.sql

```diff
@@ -1,3 +1,23 @@
-/*Queries that provide answers to the questions from all projects.*/
+-- Find all animalswhose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] Great job adding these queries, however, you are required to list just the `name` of these animals and not the entire details so your queries should start with `SELECT name ...`